qiling/os/windows/utils.py

```diff
@@ -46,3 +46,17 @@ def is_file_library(string):
 
 def string_to_hex(string):
     return ":".join("{:02x}".format(ord(c)) for c in string)
+
+
+def find_size_function(ql, func_addr):
+    # We have to retrieve the return address position
+    code = ql.mem.read(func_addr, 0x100)
+    if b"\xc3" in code:
```

Comment:
loop or something ?since its all the same

Reply:
3dd389f should resolve the issue